### PR TITLE
I have added Robots Tag

### PR DESCRIPTION
**Pull request by yuktagautam**
**What's has been added?** 

I added...

**Why is it good for SEO?**

It is good because...

**Checklist**

Please make sure you got all of these three points in your PR, otherwise the PR will be rejected.

* Details about the addition  [?]
* An example about usage  [?]
* Description of why it's good for SEO [?]

:warning:  If you have made a PR with all of these points, then *don't* include this section upon creation!

### CODE REVIEW

File: HTMLElements/README.MD [hunk 1]
```diff
@@ -84,6 +84,16 @@ Twitter has its own `<meta>` tags that are similar to the Open Graph protocol, b
 
 <br>
 
+## Use Robots Tag
+It informs search engines which pages on your site should be indexed and which should not. Can use it to save your rankings in case you have to block some pages from indexing.
+<meta name="robots" content="index, follow">
+## Use of Robots Tag
```
Comment (PatricNox) [inline HTMLElements/README.MD]: use ### and a empty newline above it

Comment (yuktagautam) [reply HTMLElements/README.MD]: I have updated it now.

File: HTMLElements/README.MD [hunk 2]
```diff
@@ -84,6 +84,25 @@ Twitter has its own `<meta>` tags that are similar to the Open Graph protocol, b
 
 <br>
 
+
+### Use Robots Tag
```
Comment (PatricNox) [inline HTMLElements/README.MD]: Sorry, I might've been unclear.

I meant for "Use of Robots Tag's Parameters" to be ### and "Use Robots tag" as ##

Comment (yuktagautam) [reply HTMLElements/README.MD]: I have updated ### to ## in Use Robots tag.

**Comment by PatricNox** [comment]
Thank you @yuktagautam  for this! :)

**Pull request merged**